utils/apps.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {AppBinding} from 'mattermost-redux/types/apps';
+
+export function fillBindingsInformation(binding: AppBinding) {
```

Comment:
Is this something that we need to rely on the client to set/assert, or is this something that needs to generated/validate on the PP/app side? (like setting the parent binding appId on all the contained bindings, right location etc.)

Reply:
I will let @levb to give take the final call here, but the idea is that all this is taken care by the client, so no redundant information is passed "through the wire" to the user agents.